api/src/main/java/org/openmrs/api/db/OrderDAO.java

```diff
@@ -269,4 +270,40 @@
 	 * @see org.openmrs.api.OrderService#getOrderGroupsByEncounter(Encounter)
 	 */
 	public List<OrderGroup> getOrderGroupsByEncounter(Encounter encounter) throws DAOException;
+	
+	/**
+	 * @see  org.openmrs.api.OrderService#getOrderGroupAttributeByUuid(String)
+	 */
+	public OrderGroupAttribute getOrderGroupAttributeByUuid(String uuid) throws DAOException;
+
+	/**
+	 * @see org.openmrs.api.OrderService#getOrderGroupAttributeTypes()
+	 */
+	public List<OrderGroupAttributeType> getAllOrderGroupAttributeTypes()throws DAOException;
+
+	/**
+	 * @see org.openmrs.api.OrderService#getOrderGroupAttributeTypeById(Integer)
+	 */
+	public OrderGroupAttributeType getOrderGroupAttributeType(Integer id)throws DAOException;
```

Comment:
Should the method name end with ById?

Reply:
Did you see the above comment?

Reply:
@dkayiwa  i think am failing to understand what you need here , however i changed the method name by removing the suffix  " ById" to it's new name **OrderDAO.getOrderGroupAttributeType**

Reply:
@tendomart did you look at the convention used by the rest of the openmrs code base for such methods?

Reply:
@tendomart did you see the above?

Reply:
seen it.

Reply:
@dkayiwa  i implemented this , but it's not reflecting here but reflecting elsewhere
![idea4](https://user-images.githubusercontent.com/16301386/92471122-0f4e9d00-f1e0-11ea-94bd-5f683a27537a.png)

Reply:
Similar to this **ProviderDAO.getProviderAttributeType(Integer providerAttributeTypeId);**

Reply:
I did it here  https://github.com/openmrs/openmrs-core/pull/3399/commits/b2bcf7ccc13c1ddfb0ab66e83c41efce5517712c